src/dotnet/commands/dotnet-new/CSharp_Web/project.json.template

```diff
@@ -0,0 +1,108 @@
+{
+  "userSecretsId": "aspnet-WebApplication-0799fe3e-6eaf-4c5f-b40e-7c6bfd5dfa9a",
+
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-3002702",
```

Comment:
Spoke to @Eilon, We are going to update this file to have `1.0.0` and `1.0.0-preview2-final`versions so that it just works when the final build is out. We can currently test this by adding `-*` to all the dependencies of the generated project.json file.